README.md

```diff
@@ -1,2 +1,61 @@
 # app.js
-GitHub Apps toolset for Node.js
+
+> Official GitHub Platform for GitHub Apps in Node.js
+
+## GitHub Apps
+
+`@octokit/app.js` allows to authenticate as GitHub Apps, installations, listen
+to webhooks, and respond to them using the GitHub API.
+
+### Authenticating as an App
+
+In order to authenticate as a GitHub App, you need to generate a Private Key
+and use it to sign a JSON Web Token (jwt) and encode it. 
+
+```js
+const auth = await appAuth({id, privateKey})
```

Comment:
where does the `appAuth` method come from? Can you add a usage example that starts with the `require('@octokit/app')`? It will help me to follow your thought process. Same with the `requestToken` method below